src/lib/utils/range.ts

```diff
@@ -15,7 +15,7 @@ function range({ start = 0, end, step = 1 }: RangeOptions): number[] {
   //   => Array length = (5000 - 1) / 500 = 9.998
   const arrayLength = Math.round((end - start) / limitStep);
 
-  return Array.apply(null, Array(arrayLength)).map(
```

Comment:
Seems the same, but why did lint not allow it?

Reply:
I forgot to add a note on this one. It was conflicting with the rule [`prefer-spread`](https://eslint.org/docs/rules/prefer-spread).